docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: .
       target: base
-    image: ov-wag
+    image: ghcr.io/wgbh-mla/ov-wag:pr-76
```

Comment:
Isn't `image` ignored in the presence of `build`? [From the docs](https://docs.docker.com/compose/compose-file/build/#definitions):
> When a build subsection is present for a service, Compose ignores the image attribute for the corresponding service, as Compose can build an image from source.

Reply:
Yes, that's leftover from the bootstrapping process, from when my computer refused to build the image.